app/containers/popup.tsx

```diff
@@ -30,73 +26,59 @@ const isChromeUrl = (url: string) => {
   return CHROME_URL_REGEX.test(url);
 };
 export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
-  componentWillMount() {
-    this.props.updateField("interceptStatus", "");
-    this.props.updateField("tabId", this.props.tabId);
-    this.props.updateField("tabUrl", this.props.tabUrl);
-    MessageService.getRequests(this.props.tabId, requests => {
-      this.props.updateField("requests", requests);
-    });
-
-    MessageService.getEnabledStatus(this.props.tabId, (enabledStatus: boolean) => {
-      this.props.updateField("enabled", enabledStatus);
-    });
-    this.props.updateField("errorMessage", "");
-    if (this.props.isInterceptorOn[this.props.tabId] === undefined) {
-      this.props.updateInterceptorStatus(this.props.tabId, true);
-    }
-  }
-
   isUrlInValid = (tabUrl: string) => {
     return !tabUrl || isChromeUrl(tabUrl);
   };
 
   handleClick = (_: React.MouseEvent<HTMLButtonElement>): void => {
-    if (this.isUrlInValid(this.props.tabUrl)) {
-      this.props.errorNotify(`Cannot Start Listening on ${this.props.tabUrl}`);
+    const props = this.props;
+    if (this.isUrlInValid(this.props.currentUrl)) {
+      props.errorNotify(`Cannot Start Listening on ${this.props.currentUrl}`, props.currentTab);
```

Comment:
consistently use `this.props` or `props`.
If we decide to use `this.props` in whole method, remove the unwanted const props.